composeApp/src/commonMain/kotlin/dev/upvote/presentation/product/ProductDetailScreen.kt

```diff
@@ -80,7 +81,7 @@ fun ProductDetailScreen(
                             )
                         )
                     } else {
-                        globalMutableStateFlow.update { it.copy(showAuth = true) }
+                        navController.navigate("auth")
```

Comment:
Ok but don't you need to provide some information so it knows how to get back on auth success?